app/assets/stylesheets/local/typography.scss

```diff
@@ -203,3 +207,22 @@ h1.heading-start {
   margin-top: 16px;
   margin-top: 1rem;
 }
+
```

Comment:
I've had another look through the CSS and it looks like there are some utility classes we could use instead, in `utilities.scss`. We could put those on existing elements, or `<span>` elements instead of `<p>`, if we really can't add to the existing code (for example because it's in a gem). 

If the classes there don't quite fit we can add to that file.